src/processor/reconnection-graph.service.ts

```diff
@@ -61,29 +62,37 @@ export class ReconnectionGraphService implements OnApplicationBootstrap, OnAppli
     this.logger.debug(`Updating graph for user ${dsnpUserStr}, provider ${providerStr}`);
     const dsnpUserId: MessageSourceId = this.api.registry.createType('MessageSourceId', dsnpUserStr);
     const providerId: ProviderId = this.api.registry.createType('ProviderId', providerStr);
+    const { key: jobId_nt, data: data_nt } = createGraphUpdateJob(dsnpUserId, providerId, SkipTransitiveGraphs);
+    const { key: jobId_t, data: data_t } = createGraphUpdateJob(dsnpUserId, providerId, UpdateTransitiveGraphs);
+
+    const [graphConnections, graphKeyPairs] = await this.getUserGraphFromProvider(dsnpUserId, providerId).catch((e) => {
+      this.logger.error(`Error getting user graph from provider: ${e}`);
+      this.graphUpdateQueue.add('graphUpdate', data_t, { jobId: jobId_t });
+      return;
+    });
```

Comment:
nit: with `async/await` I think it's better to use the `try/catch` model instead of mixing in non-async `.then()` or `.catch()`

Reply:
Yep i always mix this, fixed in next commit